app/lib/midnightSpanCorrector.js

```diff
@@ -32,9 +32,17 @@ function setNextCloseToTomorrow(openingTimes, status) {
   return status;
 }
 
+function correctClosedAt2359(status) {
```

Comment:
Could this be renamed to more closely match `openingSpansMidnight` e.g. `correctOpeningSpanningMidnight`